src/meta/processors/partsMan/DropSpaceProcessor.cpp

```diff
@@ -101,6 +101,22 @@ void DropSpaceProcessor::process(const cpp2::DropSpaceReq& req) {
     auto statiskey = MetaServiceUtils::statisKey(spaceId);
     deleteKeys.emplace_back(statiskey);
 
+    // 6. Delte related fulltext index meta data
+    auto ftRet = doPrefix(MetaServiceUtils::fulltextIndexPrefix());
+    if (!nebula::ok(ftRet)) {
+        auto retCode = nebula::error(ftRet);
+        LOG(ERROR) << "Drop space Failed, space " << spaceName
+                   << " error: " << apache::thrift::util::enumNameSafe(retCode);
+        handleErrorCode(retCode);
+        onFinished();
+        return;
+    }
+    auto ftIter = nebula::value(ftRet).get();
+    while (ftIter->valid()) {
+        deleteKeys.emplace_back(ftIter->key());
```

Comment:
It should be judged whether the current full-text index record is the current space, right?

Reply:
> It should be judged whether the current full-text index record is the current space, right?

An embarrassing bug. thanks. fixed